### ISSUE TITLE
how to implemented it in faster rcnn?

### ISSUE THREAD

**Issue by zqdeepbluesky** [mkocabas/focal-loss-keras#11]
@mkocabas 
Hi,thanks for your cool code.
but I have little confused about how to use it. I want implemented it in faster rcnn,should I just change the softmax into focal loss or others?can you tell me how to use it?
thanks so much.

**Comment by mkocabas** [comment]
Hi @zqdeepbluesky, 
The aim of focal loss is to improve the performance of one stage detectors.  Faster-RCNN is a two stage detector and it does hard example mining by selecting a constant number of RoIs from RPN. So, you don't need to use focal loss with Faster-RCNN.  But if you want to use it either, you can replace the classification loss with focal loss.

**Comment by zqdeepbluesky** [comment]
@mkocabas 
thanks so much for your answer.
